### ISSUE TITLE
Line transposition

### ISSUE THREAD

**Issue by nblackburn** [msipos/syntaxic#20]
Being able to move a selection up/down is a helpful function of some editors and would love to see it implemented here.

**Comment by msipos** [comment]
Agreed, that would be great!  The plugin API still lacks fine grained text editing, but that is planned for 1.0.  So, before 1.0, we'll have that.
